Makefile

```diff
@@ -396,3 +397,127 @@ image-mnist-job-test-build: ## Build container image with the MNIST job.
 .PHONY: image-mnist-job-test-push
 image-mnist-job-test-push: image-mnist-job-test-build ## Push container image with the MNIST job.
 	podman push ${MNIST_JOB_TEST_IMG}
+
+# RHOAI/ODH related resources installation
+
+##@ all-in-one
+.PHONY: all-in-one
+all-in-one:
+	@echo -e "\n ==> Installing Everything needed for distributed AI platform on OpenShift cluster \n"
+	-make delete-nfd-operator
+	-make delete-ai-platform-operator
+	-make delete-nvidia-operator
```

Comment:
I think the "delete" calls should be removed from here. You're already including a make target to delete the operators.